content/api/events-ingest.apib

```diff
@@ -81,6 +81,40 @@ Required Headers
             }
         }
 
+### Get Ingest Events [GET /v1/ingest/events/{id}]
+
+Downloads a gzip file of the ingest batch. 
+The binary of the gzip will be displayed to the response body.
```

Comment:
```suggestion
The binary of the gzip will be displayed in the response body.
```

Reply:
I like these notes on using this endpoint with cURL, but it seems odd to single out a service here. User's could use any number of HTTP clients and tools to make this API request. We should note the headers and such that are required to make this request, but I think any notes that are specific to cURL can be added in a separate "Notes" section just below the two GET requests here.